api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -285,6 +285,16 @@ message UICreateNewRoleCancelClickEvent {}
 // UICreateNewRoleViewDocumentationClickEvent is an event that can be triggered during custom role creation
 message UICreateNewRoleViewDocumentationClickEvent {}
 
+// AssistCompletionEvent is an event that is emitted when a completion occurs in the Assistant
+message AssistCompletionEvent {
+  // TotalTokens is the total amount of token used to satisfy this request
+  int64 total_tokens = 1;
```

Comment:
Any reasons not to use `unit64`? The number of tokens will never be negative.

Reply:
+1 (although in terms of wire format it's the same thing)

Reply:
🤷 Doesn't look like we use unsigned ints anywhere in these protos, even when the value is obviously non-negative, e.g. `resource_count`.